FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -592,6 +654,8 @@ void prvDefenderDemoTask( void * pvParameters )
         }
     }
 
+    /*********************** Collect device metrics. **********************/
+
     if( xStatus == true )
```

Comment:
Thanks for adding the comments. Can you please add a few more comments in the main function to explain each steps. Please see coreMQTT demo for an example https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/DemoTasks/MutualAuthMQTTExample.c#L443-L557